resources/chromium/generic_sensor_mocks.js

```diff
@@ -216,9 +206,13 @@ var GenericSensorTest = (() => {
     async reset() {
       if (!testInternal.initialized)
         throw new Error('Call initialize() before reset().');
-      await testInternal.sensorProvider.reset();
+      testInternal.sensorProvider.reset();
       testInternal.sensorProvider = null;
       testInternal.initialized = false;
+
+      // Wait for an event loop iteration to let any pending mojo commands in
+      // the sensor provider finish.
+      await new Promise(resolve => { setTimeout(resolve, 0); });
```

Comment:
Nit: this can be simplified as `await new Promise(resolve => setTimeout(resolve, 0));`